package.json

```diff
@@ -10,7 +10,7 @@
     "clean": "rimraf dist test/coverage",
     "lint": "eslint ./src/**.js",
     "coverage": "jest --coverage --config=jest.config.js",
-    "test": "jest --config=jest.config.js",
+    "test": "mocha",
```

Comment:
is this necessary?

Reply:
Nope, not even sure how that happened. Probably why webstorm was having so much trouble with tests :). Changed it back

Reply:
👍